pkg/controller/dns/external_dns_test.go

```diff
@@ -250,10 +255,19 @@ func TestInstances(t *testing.T) {
 		instances, err := instances(test.conf)
 		require.NoError(t, err)
 		if !reflect.DeepEqual(instances, test.expected) {
+			expectedObjects := make([]client.Object, 0)
```

Comment:
what's the point of these changes?

Reply:
logs out the objects from all pointers for easy debugging, since objects get obscured behind pointers without this